stellargraph/datasets/datasets.py

```diff
@@ -304,3 +305,106 @@ class AIFB(
     source="https://figshare.com/articles/AIFB_DataSet/745364",
 ):
     pass
+
+
+class MUTAG(
+    DatasetLoader,
+    name="MUTAG",
+    directory_name="MUTAG",
+    url="https://ls11-www.cs.tu-dortmund.de/people/morris/graphkerneldatasets/MUTAG.zip",
+    url_archive_format="zip",
+    expected_files=[
+        "MUTAG_A.txt",
+        "MUTAG_graph_indicator.txt",
+        "MUTAG_node_labels.txt",
+        "MUTAG_edge_labels.txt",
+        "MUTAG_graph_labels.txt",
+        "README.txt",
+    ],
+    description="Each graph represents a chemical compound and graph labels represent 'their mutagenic effect on a specific gram negative bacterium.'"
+    "The dataset includes 188 graphs with 18 nodes and 20 edges on average for each graph. Graph nodes have 7 labels and each graph is labelled as belonging to 1 of 2 classes.",
+    source="https://ls11-www.cs.tu-dortmund.de/people/morris/graphkerneldatasets/",
```

Comment:
I think the more human readable page is a slightly better source page, rather than the listing of the zip files.

```suggestion
    source="https://ls11-www.cs.tu-dortmund.de/staff/morris/graphkerneldatasets",
```